core/fake/mutation_storage.go

```diff
@@ -32,28 +32,28 @@ func NewMutationStorage() *MutationStorage {
 	}
 }
 
-// ReadRange returns the list of mutations
-func (m *MutationStorage) ReadRange(ctx context.Context, mapID int64, startSequence uint64, endSequence uint64, count int32) (uint64, []*pb.EntryUpdate, error) {
-	if startSequence > uint64(len(m.mtns[mapID])) {
+// ReadPage returns the list of mutations
```

Comment:
Maybe say 'ReadPage paginates through the list of mutations'

Reply:
good idea.